_includes/includes/template.php

```diff
@@ -16,8 +18,9 @@ function template_finish($foot) {
 
   function head($args=[]){
     // Args are title='My Page Title', css='page.css' showMenu=true/false, showHeader=true/false, foot=true/false
+    global $env;
 
-    $agent = isset($_SERVER['HTTP_USER_AGENT']) ? $_SERVER['HTTP_USER_AGENT'] : '';
+    $agent = isset($env['HTTP_USER_AGENT']) ? $env['HTTP_USER_AGENT'] : '';
```

Comment:
Per https://www.php.net/manual/en/reserved.variables.server.php:

> In addition to the elements listed below, PHP will create additional elements with values from request headers. These entries will be named `HTTP_` followed by the header name, capitalized and with underscores instead of hyphens. For example, the Accept-Language header would be available as `$_SERVER['HTTP_ACCEPT_LANGUAGE']`.

```suggestion
    $agent = isset($_SERVER['HTTP_USER_AGENT']) ? $_SERVER['HTTP_USER_AGENT'] : '';
```

So this entire file can be rolled back